### PR TITLE
Change deprecated buffer usage

### PR DESCRIPTION
**Pull request by benjamingr**
Fixes: https://github.com/mafintosh/tar-stream/issues/79

### CODE REVIEW

File: headers.js [hunk 1]
```diff
@@ -72,9 +72,7 @@ var toTypeflag = function (flag) {
 }
 
 var alloc = function (size) {
```
Comment (cjihrig) [inline headers.js]: I think you can remove `alloc()` completely and replace the only use with `Buffer.alloc(512)`.

**Comment by mafintosh** [comment]
@benjamingr note that is a major bump after merge (not sure if that's intended reading the original node.js issue)

**Comment by mcollina** [comment]
This should really use `safe-buffer` instead.

**Comment by ChALkeR** [comment]
Tracking: nodejs/node#19079

**Comment by mafintosh** [comment]
Fixed in master with a semver patch compatible fix

**Pull request closed without merging**